uxarray/grid/grid.py

```diff
@@ -2378,6 +2378,55 @@ def to_linecollection(
 
         return copy.deepcopy(line_collection)
 
+    def plot_edges(
+        self,
+        ax: Optional["cartopy.mpl.geoaxes.GeoAxes"] | None,  # noqa: F821
+        **plot_kwargs,
+    ):
```

Comment:
As mentioned by @erogluorhan, if we could design this function similar to how we set up `to_raster()`, that would be preferred to avoid conflicting names with our `.plot` accessors.

`to_raster` also doesn't directly call any `ax.plot` routines. To be consistent with that, having this method return something that can then be externally passed into `ax.plot` would work. 

Let me know if you have any ideas.

Reply:
Maybe something like that, or I'd also be open to discussing whether to support matplotlib plotting functions from our plotting API with a clear distinction that they are not the default `hvplot` functions, but that's where my thoughts are not in any maturity as of now. Thinking out loud: Maybe through a newer accessor such as `ux.mpl_plot`, or through the existing one with clear function namings such as `ux.plot.mpl_edges()`, or something else. I am curious what you all would think about these.